buf.gen.yaml

```diff
@@ -1,10 +1,10 @@
 version: v1
 plugins:
-  - name: cpp
+  - remote: buf.build/protocolbuffers/plugins/cpp:v3.20.0-1
```

Comment:
Does this generate the proto files for each language? Do they get uploaded anywhere? 

eg how would i import this into a java project?

or do we add publish steps to maven central, pypi, etc in a follow up?

Reply:
Publishing to the GitHub release and BSR for now.